docs/source/core_versions.rst

```diff
@@ -27,6 +27,11 @@ The RTL Freeze version of the core is indentified by a GitHub
 tag with the format cv32e40p_vMAJOR.MINOR.PATCH (e.g. cv32e40p_v1.0.0).
 In addition, the release date is reported in the documentation.
 
+Every following changes on the repository must be logically-equivalent to the RTL frozen version.
```

Comment:
changes -> change
I also think we need a little bit more explanation here stating at least that the equivalency does of course not apply to bug fixing or non-already-verified parameter combinations as explained below.

Reply:
done